Doc/library/math.rst

```diff
@@ -164,10 +164,12 @@ Number-theoretic and representation functions
    between *a* and *b*, relative to the larger absolute value of *a* or *b*.
    For example, to set a tolerance of 5%, pass ``rel_tol=0.05``.  The default
    tolerance is ``1e-09``, which assures that the two values are the same
-   within about 9 decimal digits.  *rel_tol* must be greater than zero.
+   within about 9 decimal digits.  *rel_tol* must be a nonnegative value.
```

Comment:
No, I don't think that nonnegative value for rel_tol does make sense.  Rather we should adjust implementation.

Reply:
Sorry, I didn't read your previous comment clearly.